py/reporter_service.py

```diff
@@ -79,6 +79,112 @@ def handle_request(self):
 class ThreadedHTTPServer(ThreadPoolMixIn, HTTPServer):
   pass
 
+#report some segments for the datastore
+def report(segments, trace, threshold_sec, report_levels, transition_levels):
```

Comment:
first thing to note is that to make this easy on myself and not change the logic of the actual reporting of segments is that i broke this function out so i could import the reporter service as a module and call this one important function